drivers/flash/Kconfig.nordic_qspi_nor

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2019, Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig NORDIC_QSPI_NOR
+	bool "QSPI NOR Flash"
+	select FLASH_HAS_DRIVER_ENABLED
+	select NRFX_QSPI
+	depends on HAS_HW_NRF_QSPI
+	help
+	  Enable support for nrfx QSPI driver with EasyDMA.
+
+if NORDIC_QSPI_NOR
+
+config NORDIC_QSPI_NOR_QE_BIT
+	int "Quad Enable bit number in Status Register"
+	default 6
+
+config NORDIC_QSPI_NOR_INIT_PRIORITY
+	int
+	default 80
+	help
+	  Device driver initialization priority.
+	  Device is connected to SPI bus, it has to
+	  be initialized after SPI driver.
```

Comment:
This is not true, this device does not use an SPI driver.

Reply:
I removed misleading part.